sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLPushDownTest.java

```diff
@@ -187,6 +187,8 @@ private static void initializeCalciteEnvironmentWithContext(Context... extraCont
             .defaultSchema(defaultSchemaPlus)
             .traitDefs(traitDefs)
             .context(Contexts.of(contexts))
+            // TODO[BEAM-8630]: change to BeamRuleSets.getZetaSqlRuleSets()
```

Comment:
We should get this from ZetaSQLQueryPlanner...

Reply:
Done.